src/pages/Facility/overview.tsx

```
@@ -148,6 +170,63 @@ export function FacilityOverview({ facilityId }: FacilityOverviewProps) {
+            </div>
+            <div className="grid grid-cols-1 gap-4 lg:grid-cols-2">
+              {pinnedLinks.map((link) => (
+                <div
+                  key={link.link}
+                  className="relative group block h-full transition-all duration-200 hover:ring-2 ring-primary-400 rounded-xl ring-offset-2"
+                >
+                  <button
+                    onClick={() => removeCustomLink(link.link)}
+                    className="absolute -top-2 -right-2 z-10 p-1 rounded-full bg-gray-100 hover:bg-red-100 text-gray-500 hover:text-red-600 opacity-0 group-hover:opacity-100 transition-opacity shadow-sm"
+                    aria-label={t("remove")}
+                  >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)